src/app/register/pages/register/register.component.ts

```diff
@@ -209,21 +212,23 @@ export class RegisterComponent implements OnInit, AfterViewInit {
 
     if (this.FormGroupStepA.valid) {
       this.personalData = this.FormGroupStepA.value.personal
+      const searchValue =
+        this.personalData.familyNames.value +
```

Comment:
if the user has no `familyNames`  is it ok to send the white space before the `givenName`?

Reply:
Hi @leomendoza123 yes, it is ok to send the space, the string get trimmed on server side.